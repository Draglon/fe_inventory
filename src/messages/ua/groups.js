const groups = {
  "Groups": {
    "layout": {
      "title": "Групи",
      "description": "Групи - опис",
    },
    "title": "Групи",
  },
};

export default groups;