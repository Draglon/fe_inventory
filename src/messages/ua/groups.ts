const groups = {
  "Groups": {
    "layout": {
      "title": "Групи",
      "description": "Групи - опис",
    },
    "title": "Групи",
    "addProduct": "Додати продукт",
  },
};

export default groups;