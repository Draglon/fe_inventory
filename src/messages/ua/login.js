const login = {
  "Login": {
    "layout": {
      "title": "Вхід",
      "description": "Вхід - опис",
    },
    "title": "Вхід",
  },
};

export default login;