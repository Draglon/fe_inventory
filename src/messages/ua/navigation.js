const navigation = {
  "Navigation": {
    "parishes": "Парафія",
    "groups": "Групи",
    "products": "Продукти",
    "users": "Користувачі",
    "settings": "Налаштування"
  },
};

export default navigation;