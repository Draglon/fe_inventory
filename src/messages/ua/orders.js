const orders = {
  "Orders": {
    "layout": {
      "title": "Замовлення",
      "description": "Замовлення - опис",
    },
    "title": "Замовлення",
  },
};

export default orders;