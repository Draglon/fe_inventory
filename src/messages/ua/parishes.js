const parishes = {
  "Parishes": {
    "layout": {
      "title": "Парафія",
      "description": "Парафія - опис",
    },
    "title": "Парафія",
  },
};

export default parishes;