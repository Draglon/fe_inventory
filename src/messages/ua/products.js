const products = {
  "Products": {
    "layout": {
      "title": "Продукти",
      "description": "Продукти",
    },
    "title": "Продукти",
  },
};

export default products;