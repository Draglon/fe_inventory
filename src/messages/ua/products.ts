const products = {
  "Products": {
    "layout": {
      "title": "Продукти",
      "description": "Продукти",
    },
    "title": "Продукти",
    "filter": {
      "type": {
        "label": "Тип:",
        "option": {
          "all": "Всі",
          "phone": "Телефон",
          "monitor": "Монітор"
        }
      },
      "specification": {
        "label": "Специфікація:",
        "option": {
          "all": "Всі",
          "phone": "Телефон",
          "monitor": "Монітор"
        }
      },
    },
    "guarantee": {
      "from": "з",
      "to": "до"
    }
  },
};

export default products;