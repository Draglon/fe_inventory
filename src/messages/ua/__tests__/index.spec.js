import ua from "..";

it("ua locale matches snapshot", () => {
  expect(ua).toMatchSnapshot();
});
