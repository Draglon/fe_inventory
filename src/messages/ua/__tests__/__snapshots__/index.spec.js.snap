// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`ua locale matches snapshot 1`] = `
{
  "Groups": {
    "addProduct": "Додати продукт",
    "layout": {
      "description": "Групи - опис",
      "title": "Групи",
    },
    "title": "Групи",
  },
  "LocaleSwitcher": {
    "label": "Вибрати мову",
    "locale": "{locale, select, ru {Російська} ua {Українська} en {Англійська} other {Unknown}}",
  },
  "Login": {
    "layout": {
      "description": "Вхід - опис",
      "title": "Вхід",
    },
    "title": "Вхід",
  },
  "Navigation": {
    "groups": "Групи",
    "parishes": "Парафія",
    "products": "Продукти",
    "settings": "Налаштування",
    "users": "Користувачі",
  },
  "Parishes": {
    "layout": {
      "description": "Парафія - опис",
      "title": "Парафія",
    },
    "title": "Парафія",
  },
  "Products": {
    "layout": {
      "description": "Продукти",
      "title": "Продукти",
    },
    "title": "Продукти",
  },
  "Settings": {
    "layout": {
      "description": "Налаштування - опис",
      "title": "Налаштування",
    },
    "title": "Налаштування",
  },
  "Users": {
    "layout": {
      "description": "Користувачі - опис",
      "title": "Користувачі",
    },
    "title": "Користувачі",
  },
  "shared": {
    "add": "Додати",
    "cancel": "Відмінити",
    "confirm": "Підтвердити",
    "create": "Створити",
    "logIn": "Увійти",
    "logout": "Вийти",
    "modal": {
      "removeParish": {
        "title": "Ви впевнені, що хочете видалити цю парафію?",
      },
      "removeProduct": {
        "title": "Ви впевнені, що хочете видалити цей продукт?",
      },
    },
    "next": "Наступний",
    "previous": "Попередній",
    "profile": "Профіль",
    "save": "Зберегти",
    "search": "Пошук",
    "settings": "Налаштування",
    "signUp": "Реєстрація",
  },
}
`;
